Config.cmake/fmiplatform.cmake

```diff
@@ -14,27 +14,56 @@
 # fmi_platform(platform) sets platform to one of the by FMI defined platforms
 # win32, win64, linux32, linux64, darwin32 or darwin64 depending on cmake
 # generator
-function(fmi_platform platform)
+function(fmi_platform platform fmi_version)
+    if(fmi_version EQUAL 1 OR fmi_version EQUAL 2)
+        set(FMI_1_OR_2 TRUE)
+    else()
+        set(FMI_1_OR_2 FALSE)
+    endif()
     if(CMAKE_HOST_WIN32) #Set to true when the host system is Windows and on cygwin.

```

Comment:
win32, maybe just win?

Reply:
I don't think that variable exists. This one checks for both win32 and win64: https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_WIN32.html